ui/Screen/ProjectCreation.svelte

```diff
@@ -149,14 +149,13 @@
     let response;
 
     try {
-      response = await create(
-        {
-          name,
-          description,
-          defaultBranch,
-        },
-        isNew ? newRepositoryPath : existingRepositoryPath
-      );
+      response = await create({
+        description,
+        defaultBranch,
+        repo: isNew
+          ? { type: NEW, name, path: newRepositoryPath }
+          : { type: EXISTING, path: existingRepositoryPath },
```

Comment:
These should come from an enum in typescript source file, otherwise the actual values gonna drift.

Reply:
Ya, that makes sense :+1: Too much of a n00b around the JS stuff still